simple.js

```diff
@@ -1381,19 +1378,22 @@ define(function(require, exports, module) {
                 // set Python's version
                 settings.set("project/python/@version", "python3");
 
-                // set pylint's flags
-                settings.set(
-                    "project/python/@pylintFlags",
-                    "-d all -e E -e F --generated-members=app.jinja_env.* --ignored-classes=SQLAlchemy --load-plugins pylint50"
-                );
-
                 // set PYTHONPATH
                 settings.set(
                     "project/python/@path",
                     "/home/ubuntu/.local/lib/python3.4/site-packages" +
                     ":/usr/lib/python3/dist-packages" +
                     ":/usr/local/lib/python3.4/dist-packages"
                 );
+
+                // set pylint's flags
+                settings.set(
+                    "project/python/@pylintFlags",
+                    "-d all -e E -e F --generated-members=app.jinja_env.* --ignored-classes=SQL,SQLAlchemy --load-plugins pylint50"
```

Comment:
```
$ tree -a -L 1 ~
/home/ubuntu
├── .bash_aliases
├── .bash_logout
├── .bashrc
├── .c9
├── .cache
├── .client50
├── .config
├── .domain50
├── .gem
├── .gemrc
├── .gitconfig
├── .gitignore
├── .gnupg
├── .heroku
├── .hgignore
├── .hgrc
├── .local
├── .npmrc
├── .nvm
├── .password50
├── .pip
├── .profile
├── .profileE
├── .prompt50
├── .ssh
├── .vimrc
├── bin
├── lib
├── sessions
└── workspace
```

We own `.client50`, `.domain50`, `.prompt50`, some contents of `.local/` (e.g., `pylint50`), `lib/` (e.g., `ide50.log`), and `bin/` (e.g., `debug50`).